### DISCUSSION TITLE
[Feature] <Custom Places>

### DISCUSSION THREAD

**Discussion by n1ghtBl00d** [immich-app/immich#10739] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'd like to be able to define a custom "place" for the Places section of the Explore tab. 

I have a cabin up in the mountains, and it isn't near enough to any town to get the photos taken up there sorted into any place. I'd like to have a way to define my own Place for them to be sorted into. I can find them on the map if I really want to, but the places tab is simpler to use in my opinion.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by steven-s1508** [comment]
I just started using Immich and I found this to be something I'm missing a lot.

For example, when I upload pictures of a theme park visit and I set the map marker, the "nearest" town will be shown as the location. Although not even the correct town is selected in some cases.
I set the marker for the Efteling park in the Netherlands right at the park entrance. Efteling is located in the city of _Kaatsheuvel_, but the selected place is _Loon Op Zand_, which is much farther away than the directly adjacent _Kaatsheuvel_. Maybe that in itself might be worth looking into.

Anyway, having the city as the place doesn't help me when I only have a list of city names and not the names of the parks I visited. Being able to add a custom marker on the map and give it a name, would solve this problem for me. Maybe these markers could also be reused later when opening the map to set the location of an image.

**Comment by multi-suggester** [comment]
I also think ability to define custom locations that we can circle in the map like home , school , campus ,work , home of my xxx ,park close to the home ,  ramen restaurant i like  ... would be a nice addition
